authorize/apis/transaction.py

```diff
@@ -72,17 +76,19 @@ def __init__(self, login_id, transaction_key, debug=True, test=False):
     def _make_call(self, params):
         params = convert_params_to_byte_str(params)
         params = urlencode(params)
```

Comment:
Changing this to `params = urlencode(params).encode('utf-8')` passes tests. Can you update?